perf/src/client.rs

```diff
@@ -103,9 +103,15 @@ async fn run(opt: Opt) -> Result<()> {
         }
     });
 
+    let drive_fut = async {
+        tokio::try_join!(
+            drive_uni(connection.clone(), opt.uni_requests, opt.upload_size, opt.download_size),
+            drive_bi(connection.clone(), opt.bi_requests, opt.upload_size, opt.download_size)
+        )
+    };
+
     tokio::select! {
-        x = drive_uni(connection.clone(), opt.uni_requests, opt.upload_size, opt.download_size) => x?,
-        x = drive_bi(connection.clone(), opt.bi_requests, opt.upload_size, opt.download_size) => x?,
+        _ = drive_fut => {}
```

Comment:
These tasks only terminate in the event of an error, so this shouldn't actually make a difference.

Reply:
yes, I changed that before I noticed that they run endlessly. My assumption was that we would do a certain amount of requests on each connection. And in this case it's correct to wait for requests on both uni and bidi streams to finish.

I think it's good to keep this change, since it might also be helpful if a non-endless running mode is added. E.g. measuring the amount of possible handshakes requires to wait for connections to end.